README.md

```diff
@@ -57,4 +57,7 @@ yarn start
 
 #Run all available unit and integration tests
 yarn test
+
+#Create skeleton for new component
```

Comment:
#Create all the necessary files/folders for a new, reusable component
yarn create-component $ComponentName